applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+	bool "Handle the DVFS $(dvfs_state) state"
+	default y
+
+if DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int "Timeout in milliseconds for the DVFS $(dvfs_state) state" if $(dvfs_timeout) != 0
+	default $(dvfs_timeout)
+	range 0 65535
+	help
+	  Timeout in milliseconds specifying time after which DVFS request related to
+	  the DVFS $(dvfs_state) state times out.
+	  A request is automatically cleared on timeout.
```

Following this https://github.com/nrfconnect/sdk-nrf/pull/16899/files#r1740838179 comment:

Alternatively, please do not include this option for states that do not use timeout:

```suggestion
if $(dvfs_timeout) != 0

config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
	int "Timeout in milliseconds for the DVFS $(dvfs_state) state"
	default $(dvfs_timeout)
	range 0 65535
	help
	  Timeout in milliseconds specifying the time after which DVFS request related to
	  the DVFS $(dvfs_state) state times out.
	  A request is automatically cleared on timeout.

endif # $(dvfs_timeout) != 0
```